pkg/build/options.go

```diff
@@ -44,3 +44,12 @@ func withBuilder(b builder) Option {
 		return nil
 	}
 }
+
+// withWd is a functional option for overriding the working directory
+// where ko is going run. This is exposed for testing.
+func withWd(wd string) Option {
```

Comment:
Can you avoid adding this by just doing [`os.Chdir`](https://golang.org/pkg/os/#Chdir) in the test? (You probably would want to defer a `Chdir` back to the result of `Getwd` to avoid messing up other tests).